ddt.py

```diff
@@ -38,6 +42,29 @@
     trivial_types = (type(None), bool, int, float, str)
 
 
+@unique
+class FormatTestName(Enum):
```

Comment:
Would that be problematic to rename this TestNameFormat instead? OI know it's a nit, but starting with the verb for verb sounds a bit fishy.

Reply:
Sorry that I actually took a shortcut here.  I tried to name it `TestNameFormat` but nose considered this as a test class and picked it up to run, causing the test failure.  What will be the name you want to use in case we have to avoid a name starting with `Test`?

Reply:
It could be someting like this:

``` py
from nose.tools import nottest

@unique
@nottest
class TestNameFormat(Enum):
    ...
```

Reply:
PS and please add `nose` to `requirements/build.txt`
I'm going to squash the requirement files at some point anyway

Reply:
Updated and you will see the changes in the next commit.  Thanks for the quick pointer regarding how to let `nose` to ignore the class.